starters/solidstart-tanstackquery-tailwind-modules/README.md

```diff
@@ -1,4 +1,4 @@
-# SolidStart-TanStackQuery-Tailwind-Modules
```

Comment:
Any reason for this?

Reply:
the ticket says: “SolidStart” is the name of the framework. Let’s change this in the README, index.tsx, api-example.tsx, and root.tsx files. @kenslachtajr please can you confirm that this is the required change?